docs/spec_draft.md

```diff
@@ -229,6 +229,8 @@ the IEEE-754 specification.
 // %result: [[6, 8], [10, 12]]
 ```
 
+&nbsp;[More Examples](https://github.com/sdasgup3/stablehlo/blob/main/stablehlo/tests/interpret_add.mlir)
+
```

Comment:
Also what do you think about turning "Examples` into a link? We don't really need to keep examples in the markdown file once we have exhaustive tests, right?

Reply:
We already thought about this and here are our thoughts:

We want to keep a short example right in the spec to demonstrate, using a trivial (no corner cases) example, the functioning of the op and ask the reader to follow the link for more detailed tests. I believe having that quick-example-view provides better experience.  Having said that, I agree that the example section of spec should be concise: One representative should be enough to capture the high level behavior unless the there are multiple behaviors like bitwise ops.

What do you think? 

Reply:
" believe having that quick-example-view provides better experience". Sounds good, let's give this a try and see how it works out.